### PR TITLE
feat(Configuration): 18 feat add ability to setup cors reponse and default to

### LINKED ISSUE #18: [FEAT] Add ability to setup CORS reponse and default to *
**Is your feature request related to a problem? Please describe.**

Because this api will be hosted on its own domain we will need to have cors to allow browsers to make the request properly.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by vCertxfiedGoat**
## Describe your changes
Added config to change cors and to adjust log level
## Issue ticket number and link
https://github.com/Carrera-Dev-Consulting/zombie-nomnom-api/issues/18
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have defined what tests cover my change and or added new test to cover change. 
- [x] I have added comments to the change and updated any relevant docs.
- [x] I have pulled the latest in the main branch and merged them ahead of time with no conflicts.

### CODE REVIEW

**Review by gxldCptRick — CHANGES_REQUESTED** [review]
Fix your title

**Review by Dean-Pesek — APPROVED** [review]
f this milo guy

**Comment by vCertxfiedGoat** [comment]
> Fix your title

Its fixed

**Review by gxldCptRick — CHANGES_REQUESTED** [review]
area should be (configuration)

**Comment by vCertxfiedGoat** [comment]
> area should be (configuration)

changed it to only configuration

File: pyproject.toml [hunk 1]
```diff
@@ -37,7 +37,8 @@ dependencies = [
     "click >= 8.1.7",
     "fastapi >= 0.115.0",
     "uvicorn >= 0.31.1",
-    "ariadne >= 0.23.0"
+    "ariadne >= 0.23.0",
+    "pydantic-settings >= 2.6.1"
```
Comment (gxldCptRick) [inline pyproject.toml:41]: Eww

**Review by gxldCptRick — APPROVED** [review]

**Pull request merged**